src/components/clusterWizard/BaremetalDiscovery.tsx

```diff
@@ -31,6 +31,8 @@ const BaremetalDiscovery: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
       enabledOlmOperatorsByName.ocs = { name: 'ocs' };
     } else {
       delete enabledOlmOperatorsByName.ocs;
+      // TODO(jtomasek): remove this once enabling OCS is moved into a separate storage step and LSO option is exposed to the user
+      delete enabledOlmOperatorsByName.lso;
```

Comment:
When CNV operator is selected( #536 ), the LSO is added automatically by backend.
The user should be aware of it. Maybe simple text explanation in the CNV popover is enough, @nirfarkas ?

Reply:
Let's not block that patch and resolve it within #536